frontend/src/pages/meditation/components/meditation-list/meditation-list.tsx

```diff
@@ -28,8 +41,10 @@ const MeditationList: React.FC<Properties> = ({
     >
       <BackButton onGoBack={handleBackButtonPress} />
       <div className={styles['list']}>
-        {mockedEntries.map((entry) => {
-          return <MeditationEntry meditationEntry={entry} key={entry.id} />;
+        {meditationEntries.map((entry, index) => {
+          return (
+            <MeditationEntry meditationEntry={entry} key={entry.name + index} />
```

Comment:
```suggestion
            <MeditationEntry meditationEntry={entry} key={entry.id} />
```

Reply:
@Liza-Veis Should I receive `id` from the backend side?

Reply:
yes

Reply:
as well as createdAt, updatedAt etc. We should return all the fields that we have now

Reply:
Done